### DISCUSSION TITLE
【灌水】你是一只抹茶小蛋糕

### DISCUSSION THREAD

**Discussion by TSWG4553** [share121/inter-knot#3183] [其他]
![20240720103702](https://github.com/user-attachments/assets/057e7536-fd38-4653-b61d-03e260f2120e)
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/2544d70667e060623d4b7702cd6bb3b9_8763505029750571668.png" alt="可琳-紧张" width="75" >

**Comment by Manbow1997** [comment]
可琳，你看我给你带了什么？当当当~丽娜小姐亲身给你做的巧克力蛋糕喔！快来吃吃看
